File: CoreJava/circle-app/src/com/techlabs/circle/Circle.java
package com.techlabs.circle;

public class Circle {

	private final float radius;
	private BorderStyleOption borderstyle;

	public Circle(float r) {
		radius = r;
	}

	public BorderStyleOption getBorderstyle() {
		return borderstyle;
	}

	public void setBorderstyle(BorderStyleOption borderstyle) {
		this.borderstyle = borderstyle;
	}

	public float getRadius() {
		return radius;
	}

	public Circle(float rad, BorderStyleOption border_style) {
		radius = rad;
		setBorderstyle(border_style);

	}

	public float calculateArea() {
		return (float) (3.14 * radius);
	}

	

}

File: C#/OOP/Circle-App/Circle-App/Circle.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Circle_App
{
    class Circle
    {
        private float _radius;
        private BorderStyleOption _borderstyle;

        public Circle(float r)
        {
            _radius = r;
        }

        public Circle(float rad, BorderStyleOption border_style)
        {
            _radius = rad;
            _borderstyle = border_style;
        }

        public float Radius
        {
            get
            {
                return _radius;
            }
        }
        public float calculateArea()
        {
            return (float)(3.14 * _radius);
        }

        public BorderStyleOption BorderStyle
        {
            set
            {
                _borderstyle = value;
            }
            get
            {
                return _borderstyle;
            }
        }
    }
}




